packages/cms-cli/commands/create.js

```diff
@@ -134,16 +141,19 @@ function configureCreateCommand(program) {
         return;
       }
 
-      trackCommandUsage(COMMAND_NAME, { assetType: type }, getPortalId());
+      let commandTrackingContext = { assetType: type };
 
       switch (type) {
         case TYPES.module:
           createModule(name, dest);
           break;
-        case TYPES.template:
-        case TYPES['global-partial']:
-          createTemplate(name, dest, type);
+        case TYPES.template: {
+          const { templateType } = await createTemplatePrompt();
+
+          commandTrackingContext.templateType = templateType;
```

Comment:
Is this useful? Or would it be better to just append `type = template` with the template type? (ex. `template-page`)

Reply:
@williamspiro do you have a preference here?

Reply:
As we add more wizard prompt flows, I do think tracking the options people use is useful.